src/main/java/skyler/Deadline.java

```diff
@@ -0,0 +1,45 @@
+package skyler;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents a task that needs to be done before a specific date and time
```

Comment:
Don't forget to include periods in your Javadoc comments! I noticed the same issue in other places too.